dspace-server-webapp/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -134,74 +138,76 @@ public void setUp() throws Exception {
                                  .withName("Sub Community")
                                  .build();
 
-        col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
-        col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
-        col3 = CollectionBuilder.createCollection(context, child1).withName("OrgUnits").build();
+        col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1")
```

Comment:
Although it's good to test the new scenario, it's not ok that the current setup is no longer tested

Can you restore the original tests, and add these as separate new tests instead?